use-timescale/compression/compression-design.md

```diff
@@ -80,49 +80,86 @@ because it is now just pointers to the data. When data stored like this is
+Compressing in this way is fairly effective, because timestamps are stored at
+microsecond precision and are represented as a 64-bit integer, so this allows
+the timestamp column to be stored as 64 bits for the time and the increments
+using 20 bits for a total of 26 bytes rather than 48 bytes, so the compression
+rate for this data is 46%.
+
+However, looking closely at the example, you can see the data is ordered by
+device ID, not by time. This can happen if you insert data into the hypertable
+in batches, but it can also create a problem for the compression algorithms. If
+the data is ordered by timestamp instead of by device ID, the compressed data
+looks like this instead:
+
+|Timestamp|Device ID|Status Code|Temperature|
+|-|-|-|-|
+|12:00:01|A|0|70.11|
+|0 seconds|B|0|69.70|
+|1 second|A|0|70.12|
+|0 seconds|B|0|69.69|
+|1 second|A|0|70.14|
+|0 seconds|B|4|69.70|
+
+This compresses the timestamps to 64 bits plus 2 * 20 bits, which is 73%
+compression, and much higher than the first example, for exactly the same data.
+
+By default, TimescaleDB orders data by time, but you can also add additional
+columns for ordering.
+
+## Segmenting compressed data
+
+When you have compressed your data in this way, the database has no way of
+knowing which rows need to be fetched and decompressed to resolve a query. For
+example, the database can't easily determine which rows contain data from the
+past day, as the timestamp itself is in a compressed column. You don't want to
+have to decompress all the data in a chunk, or even an entire hypertable, to
+determine which rows are required. TimescaleDB automatically includes more
+information in each batch to improve query performance.
+
+For order-by columns, TimescaleDB automatically adds additional data that stores
+the minimum and maximum value of that column. This way, the query planner can
+look at the range of timestamps in the compressed column, without having to
+decompress a batch, and determine whether the data in the batch could possibly
 match the query.
 
-When you compress your hypertable, you can also choose to specify a `SEGMENT BY`
-column. This allows you to segment compressed rows by a specific column, so that
-each compressed row corresponds to a data about a single item such as, for
-example, a specific device ID. This further allows the query planner to
-determine if the row could possibly match the query without having to decompress
-the column first. For example:
+However, using the previous example, if you wanted to query the data based on
+device ID, you would have to decompress the device column to decide if that
+batch should be part of the query. When you compress your hypertable, you can
+choose to specify a `SEGMENT BY` column. This allows you to segment compressed
+rows by a specific column, so that each compressed row corresponds to a data
+about a single item such as, for example, a specific device ID. For example, you
+could use `SEGMENT_BY` to create batches like this:
```

Comment:
Consistent? Still not sure if you want to use `SEGMENT BY` or "segment-by" in the text.

```suggestion
could use `SEGMENT BY` to create batches like this:
```